backend/internal_assembler/symbol_entry.mli

```diff
@@ -48,8 +48,8 @@ val get_bind : t -> bind
 
 val get_type : t -> symbol_type
 
-val get_shndx : t -> Owee.Owee_buf.u16
+val get_shndx : t -> Compiler_owee.Owee_buf.u16
 
 val get_name_str : t -> string
 
-val write : t -> Owee__Owee_buf.cursor -> unit
+val write : t -> Compiler_owee.Owee_buf.cursor -> unit
```

Comment:
(Certainly correct, but gave me pause:
shouldn't a module path with `__` be a
code smell?)

Reply:
Yeah, this was just missed during code review.